internal/signalfx-agent/pkg/core/config/config.go

```diff
@@ -104,7 +104,7 @@ type Config struct {
 	// Configuration of the datapoint/event writer
 	Writer WriterConfig `yaml:"writer"`
 	// Log configuration
-	Logging LogConfig `yaml:"logging" default:"{}"`
+	Logging LogConfig `yaml:"debug" default:"{}"`
```

Comment:
Since this is the signalfx agent should it be updated in this change? (I might be missing something)

Reply:
Probably not? Then again, tests don't seem to fail...

Reply:
agree let's undo this change, and maybe this points to us being able to delete this code.

Reply:
```suggestion
	Logging LogConfig `yaml:"logging" default:"{}"`
```